### ISSUE TITLE
Override the response append function

### ISSUE THREAD

**Issue by philliparente** [pklauzinski/jscroll#115]
I miss the possibility to override or parametrize the append into document function. If my response is a JSON I can't use this plugin for my needs. I would like to override, treat and do whatever I want to do with the response. What do you think?

**Comment by pklauzinski** [comment]
@philliparente yes that is a good idea, and is planned for the next version. Thanks!

**Comment by jkeam** [comment]
Thoughts on when the next version is due to be released?
